cscs-checks/microbenchmarks/likwid/likwid.py

```diff
@@ -0,0 +1,176 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class BandwidthBase(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.build_system = 'Make'
+        self.build_system.flags_from_environ = False
+        self.sourcesdir = 'https://github.com/RRZE-HPC/likwid.git'
```

Comment:
Would it make sense to use `likwid-bench` from the installed version (to make sure it works fine) instead of building likwid ? It would make the reframe check faster too.

```
module load daint-gpu
module load likwid/4.3.3-perf_event

which likwid-bench
/apps/daint/UES/jenscscs/6.0.UP07/gpu/easybuild/tools/software/likwid/4.3.3-perf_event/bin/likwid-bench
```